### DISCUSSION TITLE
[Feature Request] Custom Status/Progress

### DISCUSSION THREAD

**Discussion by silent6trinity** [Syslifters/sysreptor#235] [General]
It would be nice to be able to add a custom step in the "status" of findings/templates etc. Like instead of "In progress", it can be "awaiting xyz" or whatever. 

I've tried adjusting this myself in the actual code, but nothing seems to change.

**Comment by aronmolnar** [comment]
Yes, this is currently not possible.
We put it to our backlog for the next feature prioritization. This feature would be suitable as a feature for the pro version.
